homeassistant/components/esphome/cover.py

```diff
@@ -41,9 +41,10 @@ class EsphomeCover(EsphomeEntity[CoverInfo, CoverState], CoverEntity):
     @property
     def supported_features(self) -> CoverEntityFeature:
         """Flag supported features."""
-        flags = (
-            CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE | CoverEntityFeature.STOP
-        )
+        flags = CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE
+
+        if self._static_info.supports_stop:
```

Comment:
Hey @balloob, do you know what the value of `supports_stop` here would be, in cases when this new version of HA is running alongside an old version of ESPHome?

I think that as-written, there's a risk that the stop feature will be disabled for all ESPHome covers (even those that already support it), just because those old versions would not be sending `supports_stop = True`.

For correct backwards compatibility, we would need to treat a missing `supports_stop` value as `True`

Reply:
@jesserockz what is the default for `supports_stop` in the library in case the device doesn't share it?

Reply:
It will be false if an older device does not send the value

Reply:
https://github.com/esphome/aioesphomeapi/blob/682138b2a93dc736ea445e79ea04c0abc190873d/aioesphomeapi/model.py#L164

Reply:
So I guess I should flip that to `supports_stop: bool = True` before shipping this, right?

Reply:
I have bumped the api version so that we dont care to check that flag if an older esphome device is used with a cover.

Reply:
Ah, interesting approach. Is there any risk for other non-ESPHome consumers of the API?

Thanks for putting this together. As you can tell, I'm still learning my way around :)